source/_integrations/nanoleaf.markdown

```diff
@@ -24,6 +24,7 @@ light:
   - platform: nanoleaf
     host: 192.168.1.10
     token: xxxxxxxxxxxxxxxxxxxxx
+    name: bedroom_triangles
```

Comment:
```suggestion
```
We always try to keep the first example as minimal as possible, so only the required options are used.

Reply:
Ok, makes sense. I've removed the name from the first example, and made a second example with the name, but it's pretty obvious so I can drop it entirely if it's not adding any value.